metadata/android/de-DE/full_description.txt

```diff
@@ -0,0 +1,6 @@
+Alle benötigten Buchstaben und Zeichen sind von einer einzigen Ansicht aus erreichbar. Strg, Alt, Escape und viele weitere stehen ebenfalls zur Verfügung.
```

Comment:
This is nice, thanks! However, I won't be able to maintain these (I'll improve the `full_description` soon and there's new changelogs to write all the time).
Would it make sense to not translate the changelogs and the full description (German readers will see them in english) to be sure they are uptodate ?

Reply:
I was unsure about the changelog, but if it's not too much, I thought I could keep things up-to-date.